index.js

```diff
@@ -58,6 +58,49 @@ function message(target, context, message, isBot) {
             client.say(target, `/me Esperando um desafiante <> Expecting a challenger`)
         }
     }
+    
+    if (String(commandName).includes("!ban")) {
```

Comment:
melhor usar o startsWith pra ficar mais safe?

```suggestion
    if (String(commandName).startsWith("!ban")) {
```

Reply:
Vou corrigir isso dae.